tasks/broad/Alignment.wdl

```diff
@@ -32,6 +32,7 @@ task SamToFastqAndBwaMemAndMba {
     Int compression_level
     Int preemptible_tries
     Boolean hard_clip_reads = false
+    Boolean unmap_contaminant_reads = true
```

Comment:
If set to true this does not change any default behavior. People can set it to false for maximum concordance with hardware DRAGEN, but it is a sensible step and we recommend leaving it on

Reply:
Also I had it set at every level for backwards compatibility, if e.g. users and/or other workflows in this repo would call a subworkflow WDL, this would break because the new option hasn't been set. I changed all the added options now to be only set at the WholeGenomeGermlineSingleSample WDL, let me know if I should change it back @jessicaway

Reply:
Actually, that is exactly what happened as the womtool validation for ExomeReprocessing and ExternalExomeReprocessing are failing - perform_bqsr (also an added option that I had previously set a default value on all subworkflow levels) wasn't defined as an input in ExomeGermlineSingleSample.wdl

Reply:
sometimes it makes sense to thread it all the way through (in which case you will have a default for every top level pipeline, or you have a default at each level that a pipeline uses directly, which would allow backwards compatibility without changing the top level pipeline). If you think it will rarely be changed, or only changed by very well-informed users, you can set it at the task level input with a default. In this case you could still change it from the inputs json, but you have to know that it is an option to change.

I think either decision is reasonable and I can help look at this closer if you like.

Reply:
@jessicaway Yes, I think this case is somewhat special, because on one side we expect this option to be changed frequently (e.g. whether to use BWA mem or Dragmap is a pretty fundamental difference), so we would certainly want to be able to set that at the top level WholeGenomeSingleSample WDL. On the other side, we don't want to break backwards compatibility for all the existing pipelines out there. So I think that defaulting it at each level (to the old value) makes the most sense in this case, but please let me know if you have a better option!

Reply:
That's fine with me